packages/gatsby/cache-dir/loader.js

```diff
@@ -346,6 +346,11 @@ const queue = {
           getResourceModule(page.componentChunkName),
           getResourceModule(page.jsonName),
         ]).then(([component, json]) => {
+          if (!(component && json)) {
```

Comment:
does this ever happen?

Reply:
yes, sometimes when offline, which causes the "white page of death" error

Reply:
e.g. go to https://next.gatsbyjs.org/, clear storage (dev tools Application tab), reload, go offline (Network tab), click on plugins

Reply:
shouldn't `getResourceModule` reject/throw when it can't find stuff instead of returning null? It might be nicer to read